docs/kqueen.cli_azure_examples.rst

```diff
@@ -0,0 +1,292 @@
+Azure CLI operations with KQueen
+--------------------------------
+
+In KQueen, you can perform a number of Azure operations using the command-line interface (CLI).
+For example, you can create an Azure account, create a new Kubernetes cluster (AKS), download the Kubernetes configuration file and use it to push an application into Kubernetes.
+
+
+Configure Azure to work with KQueen
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Log in to your Azure portal:
+
+.. code-block:: bash
+
+   $ az login
```

Comment:
should we run this command inside kqueen shell or special package should be installed separately?

Reply:
That's a normal shell command... You can run it wherever you want...